controllers/enrollmentkeys.go

```diff
@@ -34,35 +34,45 @@ func enrollmentKeyHandlers(r *mux.Router) {
 //			Responses:
 //				200: getEnrollmentKeysSlice
 func getEnrollmentKeys(w http.ResponseWriter, r *http.Request) {
-	user, err := logic.GetUser(r.Header.Get("user"))
+	keys, err := logic.GetAllEnrollmentKeys()
 	if err != nil {
-		logger.Log(0, r.Header.Get("user"), "failed to fetch user: ", err.Error())
+		logger.Log(0, r.Header.Get("user"), "failed to fetch enrollment keys: ", err.Error())
 		logic.ReturnErrorResponse(w, r, logic.FormatError(err, "internal"))
 		return
 	}
-	keys, err := logic.GetAllEnrollmentKeys()
+	// handle masteradmin non-logged-in user
```

Comment:
if NetUserSecurityCheck was used as middleware vice SecurityCheck, this logic should not be required.